judgments/utils.py

```diff
@@ -244,3 +244,15 @@ def formatted_document_uri(document_uri: str, format: Optional[str] = None) -> s
         url = url + "/data.html"
 
     return url
+
+
+def linked_doc_url(document: Document):
+    press_summary_suffix = "/press-summary/1"
+    if document.document_noun == "press summary":
+        return document.uri.removesuffix(press_summary_suffix)
+    else:
+        return document.uri + press_summary_suffix
+
+
+def linked_doc_title(document: Document):
```

Comment:
can we say that document is in fact a Press Summary? I think this is more specific than a Document. I'll need to take a look at how this winds up getting used.

Should these be on the Document class; or PressSummary and Judgment classes?

Reply:
Arguably the linked-doc of a Judgment should have this prefix added.

Reply:
I tried to tie these helpers more closely to the classes, but got a bit tangled (and arguably, there's a lot more utils that should probably be on a class instance rather than a util). I've now tweaked this so both helpers support PressSummary and Judgment classes.

Reply:
I might do some work on this this afternoon in a seperate PR - I've got some ideas

Reply:
I'll be happier when this is gone, but happy for now. :)

Reply:
I’ve got something in the pipeline that should tidy a lot of this up